state/execution_test.go

```diff
@@ -13,136 +13,132 @@ import (
 func TestProposeBlock(t *testing.T) {
 	td := setup(t)
 
-	curHeight := uint32(7)
-	for i := uint32(0); i < curHeight; i++ {
-		td.moveToNextHeightForAllStates(t)
-	}
-	b1, c1 := td.makeBlockAndCertificate(t, 0, td.valKey1, td.valKey2, td.valKey3)
-	assert.NoError(t, td.state1.CommitBlock(b1, c1))
-	assert.NoError(t, td.state2.CommitBlock(b1, c1))
-	assert.Equal(t, td.state1.LastBlockHeight(), curHeight+1)
-
-	invSubsidyTx := tx.NewSubsidyTx(1, td.valKey2.Address(),
-		td.state1.params.BlockReward, "duplicated subsidy transaction")
+	proposer := td.state.Proposer(0)
+	lockTime := td.state.LastBlockHeight()
```

Comment:
`locktime` not be `LastBlockHeight + 1` ?

Reply:
Technically it can be any block number in range of `[curHeight - 8640, curHeight)`